### DISCUSSION TITLE
[Task Lists]: Make it easier to add new tasks with issues in other repositories

### DISCUSSION THREAD

**Discussion by JoshAntBrown** [community/community#5824] [Projects and Issues]
**Task lists can be pretty cumbersome to manage if you want each task to be a separate issue.** Especially if you want that issue to exist in a different repository.

Our organisation generally follows the micro-services approach and has a repository per service, projects often span a number of services including the frontend. We also have a team repository that is our initial point where we have a few issues that act as the root issue for a project, all tasks come from that in a task list. I believe many others have similar setups.

---

A while back - way before the announcement of the new task lists functionality and issues beta - I personally started working on a bit of a solution to this but ended up dropping it as building a chrome extension to add this in didn't feel like a great experience. I thought I'd share some of the early ideas from that here though in case it helps.

At the bottom of every task list is a button to add a new task.
<img width="933" alt="Screenshot 2021-09-27 at 11 46 57" src="https://user-images.githubusercontent.com/1793797/134894368-68b83d2a-ce56-4c72-8858-60a86c4a0ace.png">

Clicking that button reveals a new task workflow that allows you to quickly type in the description for the task.
<img width="936" alt="Screenshot 2021-09-27 at 11 47 47" src="https://user-images.githubusercontent.com/1793797/134894489-6056d2ff-5c70-4312-b7e9-2b5bd34fb189.png">

Adding a task through this interface would automatically create the issue in the specified repository and simply link that issue to the checklist, it could all be done via the keyboard so it was super quick to create, and manage issues from within the root issue.
 
---

The new `Convert to Issue` button is great but doesn't let you pick a repository and adds an extra step to every task you create that you want as an issue - which for us is generally all of them.

**Comment by WardenDrew** [comment]
We would also love this as a feature. Right now we have a issue tracking repository for overall QA, which then developers add entries to a task list that link to their respective repo's for their sections. Having this ability would turn it from a multi-step process (that we're seriously looking at building a chrome extension to simplify like you were) into a nice native workflow.

**Comment by WardenDrew** [comment]
Related Larger Discussion:
- https://github.com/github/feedback/discussions/4442
